app/validators/wpcc/salary_threshold_validator.rb

```diff
@@ -0,0 +1,22 @@
+module Wpcc
+  class SalaryThresholdValidator < ActiveModel::Validator
+    SALARY_THRESHOLDS  = [5876.00, 489.67, 452.00, 113.00].freeze
+    SALARY_FREQUENCIES = %w[year month fourweeks week].freeze
+    LOW_SALARY_THRESHOLDS = SALARY_FREQUENCIES.zip(SALARY_THRESHOLDS).to_h
```

Comment:
not sure if I follow @Tr4pSt3R ? The values are set in the two lines above aren't they?
```
SALARY_THRESHOLDS  = [5876.00, 489.67, 452.00, 113.00].freeze
SALARY_FREQUENCIES = %w[year month fourweeks week].freeze
```
where would the duplication occur?